srcalc/src/test/java/gov/va/med/srcalc/web/view/admin/EditRiskModelTest.java

```diff
@@ -49,12 +55,31 @@ public final void testEditRiskModel()
         
     	editRiskModel.setModelName( "NewModelName" );
     	
-        assertEquals( editRiskModel.getModelName(), "NewModelName" );
-        assertEquals( editRiskModel.getMaxDisplayNameLength(), DisplayNameConditions.DISPLAY_NAME_MAX );
+        assertEquals( "NewModelName", editRiskModel.getModelName() );
+        assertEquals(  DisplayNameConditions.DISPLAY_NAME_MAX, editRiskModel.getMaxDisplayNameLength() );
```

Comment:
Looking at it now, I don't think there is any need for the getMaxDisplaynameLength(). We have DisplayNameConditions because there are specified constraints for display names and the code should always point to that.